dll/win32/credui/lang/credui_Ro.rc

```diff
@@ -24,12 +25,12 @@ LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 {
-    IDS_TITLEFORMAT     "Conectare la „%s”"
-    IDS_MESSAGEFORMAT   "Conectarea la „%s”"
-    IDS_INCORRECTPASSWORDTITLE  "Autentificare eșuată"
-    IDS_INCORRECTPASSWORD       "Verificați corectitudinea numelui\nși a parolei de utilizator."
-    IDS_CAPSLOCKONTITLE "Modul «Caps Lock» este activ"
-    IDS_CAPSLOCKON      "Din cauza modului «Caps Lock» activ, riscați introducerea incorectă a parolei.\n\nInainte de a introduce parola, apăsați «Caps Lock» pentru a dezactiva scrierea cu majuscule."
+    IDS_TITLEFORMAT     "Conectare la %s"
+    IDS_MESSAGEFORMAT   "Conectare la %s"
+    IDS_INCORRECTPASSWORDTITLE  "Log on nu a reuşit:"
+    IDS_INCORRECTPASSWORD       "ReactOS nu vă acceptă Log on.\nAsiguraţi-vă că numele de utilizator şi\nparola sunt corecte."
+    IDS_CAPSLOCKONTITLE "Tasta Caps Lock activă:"
+    IDS_CAPSLOCKON      "Dacă aveţi tasta Caps Lock activată există posibilitatea să\nintroduceţi parola incorect.\n\nDezactivaţi tasta Caps Lock\n \înainte de a introduce parola."
```

Comment:
```suggestion
    IDS_CAPSLOCKON      "Dacă aveţi tasta Caps Lock activată există posibilitatea să\nintroduceţi parola incorect.\n\nDezactivaţi tasta Caps Lock\nînainte de a introduce parola."
```
The space after the `\n` was defintely wrong, in the meaning of wasted bytes.
And I don't think you intended to use `\î` as control character.